pkgs/development/python-modules/powerline/default.nix

```diff
@@ -37,6 +37,10 @@ buildPythonPackage rec {
 
     install -dm755 "$out/share/tmux"
     install -m644 "powerline/bindings/tmux/powerline.conf" "$out/share/tmux/powerline.conf"
+    
+    install -dm755 "$out/share/fish/vendor_functions.d"
+    install -m644 "powerline/bindings/fish/powerline-setup.fish" "$out/share/fish/vendor_functions.d/powerline-setup.fish"
```

Comment:
```suggestion
    installShellCompletion powerline/bindings/fish/powerline-setup.fish
```
... else where
```
nativeBuildInputs = [ installShellFiles ];
```

Reply:
I had seen that, however `installShellCompletion` it can't be used in this case. It has to be installed to vendor_functions.d and not vendor_completions.d. Powerline provides a function not a completion. I don't see any helper functions that target functions.

Reply:
ah, I see. okay then